xbmc/music/MusicDatabase.cpp

```diff
@@ -10231,6 +10231,48 @@ bool CMusicDatabase::GetArtTypes(const MediaType &mediaType, std::vector<std::st
   return false;
 }
 
+std::vector<std::string> CMusicDatabase::GetAvailableArtTypesForItem(int mediaId,
+  const MediaType& mediaType)
+{
+  std::vector<std::string> result;
+  if (mediaType == MediaTypeArtist)
+  {
+    CArtist artist;
+    if (GetArtist(mediaId, artist))
+    {
+      /// @deprecated artwork: fanart stored separately, doesn't need to be
```

Comment:
Better as  a doxygen `//! @todo` 
Maybe raise a roadmap issue too for artist fanart to be handled along with all the other types of art, so it does not get forgotten

Reply:
This has been on my v19 list for some time, so I won't forget, but a roadmap item is not a bad idea. The video library also has this fanart / all-other-images split for most media types.